app/1.0/blog/2016-10-03-1.7-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  "1.7 Release"
+
+---
+_We announce significant releases here on the blog, highlighting the major new features as well as breaking changes. As always, we'd love to get your feedback, issues, and PR's, so tweet us [@polymer](https://twitter.com/polymer) or contribute to the [Github](https://github.com/Polymer)._
+
+We're pleased to announce the release of:
+
+## Polymer 1.7
+
+The primary goal of the upcoming Polymer 2.0 release is to take advantage of native, cross-browser support for Web Components. With this in mind, Polymer 1.7.0 provides forwards compatibility and initial support for migration. 
+
+### `<slot>` element 
+
+Elements can opt-in to using the new `<slot>` element and `::slotted()` CSS selector from ShadowDOM v1.
+
+This is not required for current Polymer 1.x elements, but using the new syntax will make it easier to port elements to 2.x.
+
+### `html` selector in `<style is="custom-style">`, `:host > * ` selector in elements
+
+Polymer 2.x will remove the `:root` selector and replace it with `html` in `<style is="custom-style">`, and `:host > *` in element styles.
```

Comment:
It's unclear reading this if remove means removing support for `:root` or doing a replacement for you. I wonder if we can improve clarity there.